### PR TITLE
[radar-integration] Fix MP url

### PR DESCRIPTION
**Pull request by yatharthranjan**
**Description of the change**

MP url was incorrect in the redcap integration helm chart (did not contain the trailing /). This updates the mp url.

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist**

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [<name_of_the_chart>])

### CODE REVIEW

**Review by keyvaann — APPROVED** [review]
LGTM

File: charts/radar-integration/values.yaml [hunk 1]
```diff
@@ -166,8 +166,8 @@ networkpolicy:
 oauth_client_id: radar_redcap_integrator
 # -- OAuth2 client secret
 oauth_client_secret: secret
-# -- URL of the Management Portal
-managementportal_url: http://management-portal:8080/managementportal
+# -- URL of the Management Portal. Make sure to include the trailing slash
+managementportal_url: http://management-portal:8080/managementportal/
```
Comment (mpgxvii) [inline charts/radar-integration/values.yaml:170]: Should I update `radar-integration` to just include the slash in the request itself?

**Review by mpgxvii — APPROVED** [review]
LGTM, thank you.

File: charts/radar-integration/values.yaml (hunk 1 above)
Comment (yatharthranjan) [reply charts/radar-integration/values.yaml:170]: yes ideally we should use functions to concat parts of the url so less chance of these kind of error (then would work with and without the `/`)

**Pull request merged**